### DISCUSSION TITLE
Repositories Files Date UI Not proper Aligned

### DISCUSSION THREAD

**Discussion by anishkumar127** [community/community#56250] [Repositories]
### Select Topic Area

Bug

### Body

Repositories Files Date UI Design Not proper Aligned. Issue occur when Refresh Page. 
![Screenshot (755)](https://github.com/community/community/assets/77631750/79321e94-567a-4df5-be2f-eb3b63f765b3)

**Comment by dushmanta05** [comment]
Hi, You can reach out to GitHub support for this issue or wait a day; they'll update it eventually. I saw a similar bug for alignment issues last week, and the next day they updated it.

Also, I'm just curious: can we see the actual date of commit and time in the repository like in your screenshot?
